### PR TITLE
Special tooltip content when pending balance is negative

### LINKED ISSUE #6051: Simplify deposits UI - tooltip when pending balance is negative
In https://github.com/Automattic/woocommerce-payments/issues/5724, we show special message and link when available balance is negative. Turns out, pending balance can also be negative and we do not have special explanation in the tooltip when that happens.

This issue is to address that problem.

I think when pending balance is negative, its tooltip should say the same thing as the available balance tooltip when available balance is negative, ie:
"Learn more about why your account balance may be negative." where "Learn more" links to https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance

### PR DESCRIPTION
**Pull request by shendy-a8c**
Fixes https://github.com/Automattic/woocommerce-payments/issues/6051

#### Changes proposed in this Pull Request

- Show different tooltip content when pending balance is negative. I use the same tooltip content from available balance's tooltip when available balance is negative.
- Add a unit test and along with that, remove the use of `documentationUrls` in the unit test. This is so when there is a change in `documentationUrls` strings, it's detected in the unit test.

#### Testing instructions

* Make a large purchase. Can be cheap product but in large quantity.
* Refund the order.
* Notice the pending balance is negative.
* Click on the tooltip with [a link to the docs explaining about negative balance](https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance).

Screenshot 2023-05-11 at 02 41 42

* With base branch, the tooltip stays the same even though pending balance is negative.

Screenshot 2023-05-11 at 02 43 57

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6260` or branch name `add/6051-tooltip-negative-pending-balance` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/6051-tooltip-negative-pending-balance) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 781f2e73f211aadf43008e4eef30a8a163e6a39b
- Build time: 2023-05-12 02:17:01 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/add-6051-tooltip-negative-pending-balance [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Shows link to documentation in the tooltip when pending balance is negative.
```
Comment (Jinksi) [inline changelog/add-6051-tooltip-negative-pending-balance]: I've suggested a minor change to the changelog entry to improve grammar:
```suggestion
Show a link to the documentation in the tooltip when the pending balance is negative.
```

**Review by Jinksi — APPROVED** [review]
LGTM @shendy-a8c 🙌 

I've tested the following with a multi-currency WCPay account:

- [x] Selecting a currency tab with a negative pending balance shows the new tooltip content with a link to https://woocommerce.com/document/woocommerce-payments/fees-and-debits/account-showing-negative-balance/
   Screenshot 2023-05-12 at 08 47 05

- [x] Selecting a currency with a positive/zero pending balance shows the existing tooltip
   Screenshot 2023-05-12 at 08 47 19

File: changelog/add-6051-tooltip-negative-pending-balance (hunk 1 above)
Comment (shendy-a8c) [reply changelog/add-6051-tooltip-negative-pending-balance]: Thanks!

**Pull request merged**